docs/HOWTO-bn.md

```diff
@@ -1,34 +1,36 @@
+* [গিটহাব হ্যালো ওয়ার্ল্ড](https://docs.github.com/en/get-started/quickstart/hello-world) *(in english)*
+* [ইউটিউব - গিটহাব টিউটোরিয়াল প্রাথমিকদের জন্য](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in english)*
+* [ইউটিউব - গিটহাব রিপোজিটরি ফর্ক করা এবং পুল রিকোয়েস্ট সাবমিট করা](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(in english)*
+* [ইউটিউব - মার্কডাউন ক্র্যাশ কোর্স](https://www.youtube.com/watch?v=HUBNt18RFbo) *(in english)*
 
 
-কোন প্রশ্ন করতে দ্বিধাবোধ করবেন না। সব কন্ট্রিবিউটরই প্রথম PR থেকে শুরু করেছিল। তাহলে কেন না আমাদের [large, growing](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) community তে যোগদান করুন!
+প্রশ্ন করতে দ্বিধাবোধ করবেন না; সমস্ত কন্ট্রিবিউটর তাদের প্রথম PR দিয়ে শুরু করেছিলেন। তাহলে কেন না আমাদের বড় এবং বাড়তি সংখ্যক সম্প্রাণ্ডে যোগদান করতে চান। [community](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) তে যোগদান করুন!
 
 <details align="center" markdown="1">
-<summary>Click to see the growth users vs. time graphs.</summary>
+<summary>ব্যক্তিগত সময় সাথে ব্যবহারকারীদের প্রবৃদ্ধি দেখতে ক্লিক করুন।</summary>
 
 [![EbookFoundation/free-programming-books's Contributor over time Graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)
 
 [![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)
 
 </details>
 
-এমনকি আপনি যদি একজন অভিজ্ঞ ওপেন-সোর্স কন্ট্রিবিউটর ও হয়ে থাকেন, কখনও কখনো কিছু জিনিস ভুল হতেই পারে। যখন আপনি আপনার PR সাবমিট করবেন ***GitHub Actions* আপনার কোড কে যাচাই-বাছাই করবে, কখনো বা স্পেসিং বা ক্যাপিটালাইজেশন এর মত ছোটখাটো জিনিস খুঁজে বের করবে**। যদি আপনি সবুজ বাটন পেয়ে যান, তাহলে বুঝতে পারবেন সবকিছু রিভিউ এর জন্য প্রস্তুত। কিন্তু যদি আপনি সবুজ বাটন না পান তাহলে ফেইল্ড হওয়া চেক এর নিচে "Details" এ ক্লিক করলে সমস্যাগুলি খুঁজে বের করতে পারবেন। তারপর সেই সমস্যাগুলো ফিক্স করার পর আপনার PR এ কমিট করবেন।
+যদি আপনি একজন অভিজ্ঞ ওপেন সোর্স কন্ট্রিবিউটর হন, তবে আপনাকে কিছু সমস্যা দেওয়ার কথা হতে পারে। আপনি আপনার PR সাবমিট করার পর, GitHub Actions একটি লিন্টার চালাবে, স্পেসিং বা অক্ষরক্রমে সামান্য সমস্যাগুলি প্রায় সাধারণভাবে খুঁজে পেতে। যদি আপনি একটি সবুজ বোতাম পান, তাদের জন্য সমস্ত বিচারের জন্য সবকিছু প্রস্তুত আছে; কিন্তু যদি না পান, তবে যে লিন্টার চেক ব্যর্থ হয়েছে সেটির অধীন "বিস্তারিত" এ ক্লিক করে খুঁজে বের করতে এবং সমস্যাটি ঠিক করতে আপনি আপনার PR টি খোলা শাখার মধ্যে নতুন কমিট যুক্ত করতে পারবেন।
```

Comment:
The previous translation was better than this version. The previous translation clearly explained what happens after submitting a PR. I think it will be better if we revert the change for this section. The more fluent translation looks like this:

> এমনকি আপনি যদি একজন অভিজ্ঞ ওপেন-সোর্স কন্ট্রিবিউটর ও হয়ে থাকেন, কখনও কখনো কিছু জিনিস ভুল হতেই পারে। যখন আপনি আপনার PR সাবমিট করবেন ***GitHub Actions* আপনার কোড কে যাচাই-বাছাই করবে, কখনো বা স্পেসিং বা ক্যাপিটালাইজেশন এর মত ছোটখাটো জিনিস খুঁজে বের করবে**। যদি আপনি সবুজ বাটন পেয়ে যান, তাহলে বুঝতে পারবেন সবকিছু রিভিউ এর জন্য প্রস্তুত। কিন্তু যদি আপনি সবুজ বাটন না পান তাহলে ফেইল্ড হওয়া চেক এর নিচে "Details" এ ক্লিক করলে সমস্যাগুলি খুঁজে বের করতে পারবেন। তারপর সেই সমস্যাগুলো ফিক্স করার পর আপনার PR এ কমিট করবেন।